src/streamers/spotify.streamer.ts

```diff
@@ -63,11 +63,11 @@ export class SpotifyStreamer implements IStreamer {
 
     const response: Response = await fetch('https://accounts.spotify.com/api/token', { method: 'POST', headers, body: params });
     const jsonData = await response.json() as SpotifyAccessToken;
-    this.storer.set(SPOTIFY_ACCESS_TOKEN_KEY, jsonData);
+    this.storer.set(SPOTIFY_ACCESS_TOKEN_KEY, jsonData, jsonData.expires_in - 1000);
```

Comment:
Spotify returns an expiry date for the access token, so we can just use that instead of defaulting to `Infinity`